Dockerfile

```diff
@@ -9,13 +15,6 @@ ENV REPO_PATH=/managed-cluster-config
 COPY . ${REPO_PATH}
 WORKDIR ${REPO_PATH}
 
-# Docker dependencies:
-RUN yum update -y && yum install -y \
-    git \
-    make \
-    epel-release
-RUN yum -y install python-pip && yum clean all
-
 # Upgrade pip and install necessasry packages
 RUN pip install \
     --upgrade pip \
```

Comment:
Now that we are using the python 2.7 docker image I don't see why using the venv.

Couldn't we remove it and go straight to `RUN make`?